packages/airnode-node/src/types.ts

```diff
@@ -166,18 +166,24 @@ export interface EVMContracts {
 
 export interface EVMProviderState {
   readonly contracts: EVMContracts;
-  readonly gasPrice: ethers.BigNumber | null;
+  readonly gasPrice: GasTarget | null;
   readonly provider: ethers.providers.JsonRpcProvider;
   readonly masterHDNode: ethers.utils.HDNode;
   readonly currentBlock: number | null;
 }
 
 export interface TransactionOptions {
-  readonly gasPrice: number | ethers.BigNumber;
+  readonly gasPrice: GasTarget;
   readonly masterHDNode: ethers.utils.HDNode;
   readonly provider: ethers.providers.JsonRpcProvider;
 }
 
+export interface GasTarget {
```

Comment:
Maybe it's worth including the transaction type here so that there's a single way of checking what value to use?

Reply:
If GasTarget.gasPrice is `!undefined`, it's transaction type is 1, otherwise transaction type 2. 
GasTarget can actually be swapped out with `ethers.providers.FeeData` - which I'll do now.

Reply:
Ehh I guess this object is simple enough that it doesn't really matter. Was just a suggestion but I think you're right about it being a bit unnecessary